### PR TITLE
Add support for other tables:

### PR DESCRIPTION
**Pull request by kristenmills**
Well it seems like it works. But I only pretend to be good at sql so i'm not yolo-ing this one.

### CODE REVIEW

File: setup.sql [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
+-- Funtion that notifies channel with json encoded data.
+CREATE OR REPLACE FUNCTION table_change()
   RETURNS trigger AS
 $BODY$declare
-	event TEXT;
+  result TEXT;
 begin
-	select row_to_json(events) into event from events where id=new.id;
-	perform pg_notify('ssevents', event);
+  execute 'select row_to_json(' || TG_ARGV[0] || ') from ' || TG_ARGV[0] || ' where id=$1.id' into result using new;
```
Comment (7imbrook) [inline setup.sql:8]: You've verified this right?

Comment (kristenmills) [reply setup.sql:8]: Yep

Comment (7imbrook) [reply setup.sql:8]: YOLO merge?

Comment (kristenmills) [reply setup.sql:8]: YOLO

**Pull request merged**